vmdb/Gemfile

```diff
@@ -42,7 +42,13 @@ gem "acts_as_tree",                   "~>2.1.0"  # acts_as_tree needs to be requ
 # https://github.com/jeremyevans/ruby-american_date
 gem "american_date"
 gem "default_value_for",              "~>2.0.3"
-gem "thin",                           "~>1.3.1"  # Used by rails server through rack
+
+if RUBY_PLATFORM == 'java'
+  gem "puma"
+else
+  gem "thin",                           "~>1.3.1"  # Used by rails server through rack
+end
```

Comment:
@Fryguy @gtanzillo @jrafanie @tenderlove Sorry to say that I never even heard of puma.  If we are reviewing which app server to ship with, should we expand the list to other candidates?